### ISSUE TITLE
Image Style Transfer Using Convolutional Neural Networks

### ISSUE THREAD

**Issue by dltmddbs100** [dltmddbs100/Paper-log#11]
## Main Idea
**Rendering the semantic content of an image with CNN**
Paper: <https://ieeexplore.ieee.org/document/7780634>

## Abstract
Rendering the semantic content of an image in different styles is a difficult image processing task. Arguably, a major limiting factor for previous approaches has been the lack of image representations that explicitly represent semantic information and, thus, allow to separate image content from style. Here we use image representations derived from Convolutional Neural Networks optimised for object recognition, which make high level image information explicit. We introduce A Neural Algorithm of Artistic Style that can separate and recombine the image content and style of natural images. The algorithm allows us to produce new images of high perceptual quality that combine the content of an arbitrary photograph with the appearance of numerous wellknown artworks. Our results provide new insights into the deep image representations learned by Convolutional Neural Networks and demonstrate their potential for high level image synthesis and manipulation.

## Contribution
+ 이전 style transfer 방식은 low-level feature가 중심을 이뤘기 때문에 high-level feature에서는 거의 변화가 없는 모습, 여기서는 high-level feature의 semantic information도 활용
+ CNN의 deep representation에 대한 학습을 이용해 한 이미지의 style을 다른 이미지의 content와 결합하는 방식을 제시
+ 이미지가 network를 통하면서 반환하는 feature map을 추출, noise image가 이와 유사한 feature map을 갖도록 학습
+ Content loss는 high-level layer를 사용, detail한 pixel 정보는 소실되는 정보의 추상화가 발생 => styler epresentation을 high-level의 layer와 matching할 경우 local한 이미지 구조를 잘 보존해 생성되는 이미지가 더 continuous하며 smoother
+ Style loss는 서로 다른 feature map간의 상관관계로 정의, Gram matrix를 사용해 noise image의 feature map의 Gram matrix와 style image의 Gram matrix가 유사해지는 방향으로 학습
+ 다만 style loss는 content loss와 다르게 단일 layer가 아닌 pretrained network인 VGG19의 low-level부터 high-level까지 다양한 수준에서 loss 계산
+ Noise image는 계산된 feature map을 바탕으로 content loss와 style loss의 선형 결합인 total loss를 줄이는 방향으로 학습
## Reviews
<https://deepkerry.tistory.com/21>
